src/components/Message/MessageRepliesCountButton.tsx

```diff
@@ -25,17 +25,15 @@ const UnMemoizedMessageRepliesCountButton: React.FC<MessageRepliesCountButtonPro
     if (labelSingle) {
       singleReplyText = `1 ${labelSingle}`;
     } else {
-      singleReplyText = t('1 reply');
+      singleReplyText = t('replyCount_one', { replyCount: reply_count });
     }
   }
 
   if (reply_count && reply_count > 1) {
     if (labelPlural) {
       pluralReplyText = `${reply_count} ${labelPlural}`;
     } else {
-      pluralReplyText = t('{{ replyCount }} replies', {
-        replyCount: reply_count,
-      });
+      pluralReplyText = t('replyCount_other', { replyCount: reply_count });
```

Comment:
Based on what [I am seeing here](https://www.i18next.com/translation-function/plurals#singular-plural), you're not supposed to use `key_one` or `key_other` as a key. Instead, you should use just `key` and let the framework pull the right key based on the json provided and the count available.